updateCli.d/pluginsite.yaml

```diff
@@ -0,0 +1,49 @@
+source:
+  kind: githubRelease
+  spec:
+    owner: "jenkins-infra"
+    repository: "plugin-site-api"
+    token: ""
+    username: "olblak"
+    version: "latest"
+conditions:
+  docker:
+    name: "Docker Image Published on Registry"
+    kind: dockerImage
+    spec:
+      image: "jenkinsciinfra/plugin-site"
```

Comment:
Shouldn't this be the backend image, so plugin-site-api?

Reply:
Yes it should even if for now, it won't work because, jenkinsciinfra/plugin-site-api doesn't have a tag similar to github release like jenkins-wiki-exporter :)